### ISSUE TITLE
Add content from 3rd edition PDF file

### ISSUE THREAD

**Issue by zondo** [i6/ibg#1]
Up to chapter 1 so far.

**Comment by DavidGriffith** [comment]
What OS / distribution are you using?  I'm using Debian 8 and Sphinx installed from APT and got this:
```
Exception occurred:
  File "/usr/lib/python2.7/dist-packages/sphinx/util/__init__.py", line 94, in get_matching_docs
    suffixpattern = '*' + suffix
TypeError: cannot concatenate 'str' and 'list' objects
```
I suspect the cause is some sort of version problem.

**Comment by DavidGriffith** [comment]
Could you roll back the release and version stuff to the Third Edition?  I want to get the Third Edition committed and tagged before we start adding Fourth Edition stuff.

**Comment by zondo** [comment]
That's what I planned to do as well.  I've renamed the milestones to make it clearer.

**Comment by DavidGriffith** [comment]
I mentioned this because when I built the IBG, I noticed that it identified itself as Release 4.0.

**Comment by zondo** [comment]
Hmm, looks like your sphinx version is too old.  Other people had the same
problem:

https://github.com/rtfd/readthedocs.org/pull/1187

Can you update to the latest sphinx (latest version is 1.4)?

**Comment by DavidGriffith** [comment]
I deleted my comment about the build failure because, yes, it was a too-old version.  Installing with "pip install sphinx" did the trick.

**Comment by zondo** [comment]
Re versioning: sphinx projects have a 'release' number.  I arbitrarily put 4.0 since it will eventually be the 4th edition, but maybe it should be independent of the edition.

**Comment by DavidGriffith** [comment]
Please just put it at 3 (no dot) and delete the fourth edition note in the release history.

**Comment by zondo** [comment]
Done.

**Comment by zondo** [comment]
Added chapter 2.

**Comment by DavidGriffith** [comment]
The Copyright, Release History, Forward, and About This Guide need to be changed to be forward unnumbered sections.  Sphinx is treating all sections as proper chapters.  I know how to fix this in LaTeX, but not Sphinx.

**Comment by zondo** [comment]
Not quite sure what you mean here, unless you're looking at the output of 'make latex'.  In which case, the LaTeX output can be customized in 'conf.py' (settings near the end of the file).  I'm concentrating on content rather than style at the moment, but feel free to tweak those settings as you see fit.

**Comment by zondo** [comment]
Added chapters 3 and 4.

**Comment by zondo** [comment]
Added chapter 5.

**Comment by DavidGriffith** [comment]
What I mean is that in the Framemaker-generated IBG.pdf, chapter 1 is "Just what is interactive fiction?".  Before that in the table of contents are the foreword and "About this guide".  In this version, chapter numbering starts at Copyright.  What should be Chapter 1 is instead Chapter 5.  This mistake happens when building to latexpdf, but not to html.

**Comment by DavidGriffith** [comment]
I've run into two problems with Sphinx so far that cause irritating problems for accurately rendering a document into PDF: Inability to wrap text around graphics and inability to correctly number chapters.  This makes me think that Sphinx won't be stable enough for this project.

**Comment by zondo** [comment]
These are not problems with Sphinx, but just reflect the default LaTeX settings.  I haven't yet done _any_ tweaking of these; as I mentioned above, I'm concentrating on the content right now.

Your main goal at the moment seems to be (correct me if I'm wrong) to recreate the exact look of the 3rd edition PDF.  And I have no doubt that with suitable adjustment, LaTeX can do all that and more.  Sphinx leaves a hook for people to include their own LaTeX preamble, which pretty much gives complete control over everything.

As an aside, while it's nice to have a printable version, IMHO the online version is going to be the preferred way to read it for many people.  I certainly wouldn't want to be typing in the examples from a printed book!

**Comment by DavidGriffith** [comment]
I want to have this project render correctly to PDF because I would like to have it printed.  If Sphinx can't handle some basic things like word wrap and chapter number, then we're going to have to look at abandoning Sphinx for LaTeX and use a LaTeX-to-html converter.

**Comment by zondo** [comment]
As I said, these are just the default LaTeX settings!  It's not that these things can't be handled, just that they _haven't been handled yet_.  I'd rather leave the look-and-feel of things until later.

If, at the end of the conversion, you still don't like the way the printed version looks, you can always take the LaTeX generated by Sphinx and use that as the source input for your preferred version.  But I think you should at least give Sphinx a chance!

**Comment by DavidGriffith** [comment]
Could we pause adding content and investigate how to fix these problems?  I'd hate to get it complete and ultimately find out that what I'm looking for cannot be done.

**Comment by zondo** [comment]
Fair enough.  I'll finish off the bits I've been working on and push those for review, and then start looking at #7.

**Comment by zondo** [comment]
I've tweaked the LaTeX settings enough to fix #7.  There's probably more tweaks to be done (e.g., start each new section on a new page) but it was good enough to publish.

**Comment by DavidGriffith** [comment]
I got the LaTeX side using Emerald to do the woodcut drop caps.  Here's how I've been doing them:

```
.. only:: html
  .. image:: /images/picB.png
     :align: left

.. raw:: latex

    \dropcap{b}

efore we start learning to use the Inform system, it's probably sensible...
```

Thanks for pointing out that package to me.

**Comment by DavidGriffith** [comment]
I think Sphinx will work.  Is there any reason I need to keep the latex branch around?

**Comment by DavidGriffith** [comment]
In an effort to avoid needlessly provoking merges, how about you continue on adding chapters 6 through 9 and I'll start on 10 to 17, at least for the time being?

**Comment by zondo** [comment]
Sounds good.  I'll make a start on chapters 6--9 this evening.

Re the latex branch: if you're not planning to continue with it in parallel, then it can be closed down.

**Comment by zondo** [comment]
Chapters 6--9 completed.  Working on rest of the appendices now...

**Comment by DavidGriffith** [comment]
Remaining chapters 10 to 17 have been completed.  I think this concludes the transcribing phase of the project.
